PHPCompatibility/Sniffs/Classes/NewClassesSniff.php

```diff
@@ -1389,6 +1457,10 @@ private function processCatchToken(File $phpcsFile, $stackPtr)
      */
     protected function handleFeature(File $phpcsFile, $stackPtr, array $itemInfo)
     {
+        if (isset($this->importedClasses[strtolower($itemInfo['name'])])) {
```

Comment:
```suggestion
       if (isset($this->importedClasses[$itemInfo['nameLc']])) {
```

The `$itemInfo` already has the lowercased name available ;-)